frontends/ol-search-ui/src/components/LearningResourceCard.tsx

```diff
@@ -1,7 +1,7 @@
 import moment from "moment"
```

Comment:
LearningResourceCard is not really search-specific; at some point I would like to move this file (and related typedefs) to a separate package, perhaps `ol-learning-resource-ui`, so that `ol-search-ui` goes back to being more search-specific (searchbox, facets).